### DISCUSSION TITLE
Active Backends Value in JSON output?

### DISCUSSION THREAD

**Discussion by billglennon** [rapidloop/pgmetrics#48] [Q&A] [answered]
Hi,
I see the value Active Backends field and value in the output format = human. Where does this exist in the JSON output format?
Thanks!

**Comment by mdevan** [comment, marked as answer]
The whole of the JSON output is one object. It has a key called `backends`, and it's value is an array of objects. The number of elements of this array is the number of active backends. In Javascript:

`active_backends = output.backends.length;`

**Comment by billglennon** [comment]
Thanks!
